CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[5.0.0] - 2022-04-22
+--------------------
+
+* Removed LegacyWaffle* classes
```

Comment:
Reminder to search the whole repository. There is some how-to doc clean-up needed.

Reply:
Yeah, thanks
Removed the LegacyWaffle* section from the how-to doc

Reply:
Do you mind updating as follows?
```suggestion
* BREAKING CHANGE: Removed LegacyWaffle* classes. Although this is a breaking change, all known uses have already been fixed.
```

Reply:
Done!)